updating/index.adoc

```diff
@@ -0,0 +1,67 @@
+== Understanding upgrade channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels-releases[Understanding upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are specific to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install. The `openshift-install` binary file for a specific version of {product-title} always installs that minor version. In addition to the minor version, you can select from the `candidate`, `fast`, `stable` and `eus` channel tiers to enable an update strategy that suits the needs of your environment. For more information, see the following:
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and release paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#upgrade-version-paths_understanding-upgrade-channels-releases[Upgrading version paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding fast and stable channel use and strategies]
+* xref:../updating/understanding-upgrade-channels-release.adoc#restricted-network-clusters_understanding-upgrade-channels-releases[Understanding restricted network clusters]
+* xref:../updating/understanding-upgrade-channels-release.adoc#switching-between-channels_understanding-upgrade-channels-releases[Switching between channels]
+
+[id="updating-clusters-overview-update-cluster-using-web-console"]
+== Updating a cluster using the web console
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster using the web console]: You can update an {product-title} cluster by using the web console. For more information, see the following items:
+
+* xref:../updating/updating-cluster-within-minor.adoc#update-upgrading-web_updating-cluster-within-minor[Updating a cluster by using the web console]
+* xref:../updating/updating-cluster-within-minor.adoc#update-changing-update-server-web_updating-cluster-within-minor[Changing the update server by using the web console]
+
+[id="updating-clusters-overview-update-cluster-using-cli"]
+== Updating a cluster within a minor version using the CLI
+xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version using the CLI]: You can update an {product-title} cluster within a minor version by using the OpenShift CLI (oc).  For more information, see the following procedures:
+
+* xref:../updating/updating-cluster-cli.adoc#update-upgrading-cli_updating-cluster-cli[Updating a cluster by using the CLI]
+* xref:../updating/updating-cluster-cli.adoc#update-changing-update-server-cli_updating-cluster-cli[Changing the update server by using the CLI]
```

Comment:
Probably lots of precedent for "server", but personally I prefer "service" to be less opinionated about how many there are.  Plus it matches the formal "OpenShift Update Service" name.

Reply:
Ack'ed

Reply:
In fact, now you have `Changing the update service by using the web console` above and `Changing the update server by using the CLI` here.  Whichever noun we chose, we should at least be consistent over the space of a few lines in this file.